src/commands/registerCommands.ts

```diff
@@ -0,0 +1,87 @@
+import * as path from 'path';
+import { commands, ExtensionContext, Position, Uri, window, workspace } from "vscode";
+import { CancellationToken, ExecuteCommandParams, ExecuteCommandRequest, LanguageClient, ReferencesRequest, TextDocumentIdentifier } from "vscode-languageclient";
+import { markdownPreviewProvider } from "../markdownPreviewProvider";
+import { CommandConstants } from "./commandConstants";
+
+/**
+ * Register the commands for vscode-xml
+ *
+ * @param context the extension context
+ */
+export async function registerCommands(context: ExtensionContext, languageClient: LanguageClient) {
+
+  // XML Documentation commands
+  context.subscriptions.push(markdownPreviewProvider);
```

Comment:
Create a new function registerDocsCommands and move the docs code to this function